### ISSUE TITLE
Name of indicator when changed and save does not immediately update on the modal

### ISSUE THREAD

**Issue by RickieT** [mercycorps/TolaActivity#1527]
Media displaying issue: 
![react_ind_name](https://user-images.githubusercontent.com/23220180/56983139-c5f43500-6b37-11e9-8176-9166a66c56ed.gif)

**Comment by jennyhavah** [comment]
Since we'll be doing a lot of work on this modal in Satsuma, I'm moving it to that project. Seems like as good a time as any to tackle it.

**Comment by jennyhavah** [comment]
@camgnostic I'm not seeing the fix. In the screenshot below, note that the modal title hasn't refreshed to match the new name I just saved.

![image](https://user-images.githubusercontent.com/33670923/59171761-6f482680-8af9-11e9-909a-25847e8de254.png)

The steps I took were:
1. Edit the indicator and click Save.
2. Be prompted to complete the newly required Result level tab.
3. Be prompted for a rationale because there were results linked to the indicator.
4. See success message, but modal title does not refresh.
